it/nationaldutyrepaymentcenter/controllers/NDRCAmendCaseISpec.scala

```diff
@@ -18,19 +17,15 @@ import uk.gov.hmrc.nationaldutyrepaymentcenter.services.{NDRCAuditEvent, UUIDGen
 
 import java.time.{ZoneId, ZonedDateTime}
 import java.{util => ju}
+import java.time.Clock
```

Comment:
OBSERVATION: You removed the clock, but brought the import (unless it's being used somewhere I cannot see).

Reply:
Clock was already imported - but not in the header of the file.   It's used in this line
`bind[Clock].toInstance(clock),`

For some reason, overriding the `clock` definition (with exactly the same time as defined in the trait) was causing it to be null when injected into the application.